Dockerfile

```diff
@@ -0,0 +1,54 @@
+# TODO: Once stable, this should all go into the base image
+#
+# Base image at https://github.com/wordpress-mobile/docker-gb-mobile-image
+FROM public.ecr.aws/automattic/gb-mobile-image as gutenberg-test-runner
+
+WORKDIR /app
+
+ENTRYPOINT ["/bin/bash", "--login", "-c"]
+
+# Ensure the Node and npm version are the ones expected by the project
+COPY .nvmrc .nvmrc.host
+RUN nvm install $(cat .nvmrc.host)
+# Notice the old version of npm.
+RUN npm install -g npm@6
```

Comment:
See the `package.json` setting:

https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/package.json#L9-L12